docs/questions/query-builder/expressions-list.md

```diff
@@ -424,13 +429,14 @@ Related: [doesNotContain](#doesnotcontain), [regexExtract](#regexextract).
 
 ### date
 
-> Only available for PostgreSQL.
+> Not available for Oracle and then non-JDBC Apache Druid driver.
 
-Converts an ISO 8601 date string to a date. The string _must_ be in a valid ISO 8601 format.
+- When used on a string, converts an ISO 8601 date string to a date. The string _must_ be in a valid ISO 8601 format. If the string contains time, the time part is truncated.
+- When used on a datetime value, truncates datetime to a date.
 
 Syntax: `date(value)`
 
-Example: `date("2025-03-20")` would return a date value so that you can use all the date features in the query builder: group by month, filter by previous 30 days, etc.
+Example 1: `date("2025-03-20")` would return a date value so that you can use all the date features in the query builder: group by month, filter by previous 30 days, etc.
```

Comment:
```suggestion
Example: `date("2025-03-20")` would return a date value so that you can use all the date features in the query builder: group by month, filter by previous 30 days, etc.
```